### PR TITLE
[MIG][11.0] website_membership: nothing to do

### PR DESCRIPTION
**Pull request by cubells**
Depends on

- [x] membership

cc @Tecnativa

### CODE REVIEW

File: odoo/openupgrade/doc/source/modules100-110.rst [hunk 1]
```diff
@@ -565,7 +565,7 @@ missing in the new release are marked with |del|.
 +-----------------------------------+-------------------------------------------------+
 |website_mass_mailing               |                                                 |
 +-----------------------------------+-------------------------------------------------+
-|website_membership                 |                                                 |
+|website_membership                 | Nothing to do                                    |
```
Comment (pedrobaeza) [inline odoo/openupgrade/doc/source/modules100-110.rst]: The table is incorrect (one space more).

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by cubells** [comment]
@pedrobaeza 
done!

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please make `membership` migration

**Comment by cubells** [comment]
@pedrobaeza 

done: https://github.com/OCA/OpenUpgrade/pull/1364

**Pull request merged**